### DISCUSSION TITLE
Debian Package

### DISCUSSION THREAD

**Discussion by trymeouteh** [andyholmes/valent#745] [Distribution Packaging]
Please release this package as a bare metal Debian package

1. Get it into the official Debian and Ubuntu package repo
2. Always include a DEB install with every release on Github

**Comment by andyholmes** [comment]
Sorry, I really have no time to pick up Debian package maintaining :slightly_frowning_face: 

I'll probably only ever distribute a Flatpak, as there are far too many distributions and configurations for me to learn and understand. Package maintainers are in a much better position to understand the requirements and serve as first contact for bug reports.

**Comment by Maryjane-ai** [comment]
> Get it into the official Debian and Ubuntu package repo

Upstream developers can, _but_ do not have to be the persons to do the work for it to be included in the debian (or any distro) repositories. 

A first step can be to do an RFP request for package with Debian. Anyone can do that. https://wiki.debian.org/RFP

And see if a Debian Developer is interested in picking up packaging Valent.

**Comment by rrrrrrmb** [comment]
Hi, for those interested, I've packaged Valent  for Debian. However, it likely won't be included in a Debian stable release until Valent itself has a stable release.

Until then, users can download and install it from my [repository](https://download.opensuse.org/repositories/home:/rmb/Debian_Testing/). (Debian 13 Trixie/Testing)
